### ISSUE TITLE
add navigation between blocks

### ISSUE THREAD

**Issue by nloureiro** [ethproofs/ethproofs#173]
On the blocks page, add navigation to the previous and next block.

![Screen Shot 2024-12-19 10 56 17 AM](https://github.com/user-attachments/assets/f1ebfe2f-5b02-49a0-b7ac-3ac23ee92342)

figma for details
https://www.figma.com/design/HE0ZQ8ISCCuwkPy6iot9m3/ETHProofs-site?node-id=6054-71976&t=O5tbH9V4ih5ctRH4-1

**Comment by wackerow** [comment]
@nloureiro To clarify... "Next" block means, next block with an available proof? or next numerical block?

Assuming next block with an available proof; ie, the next block we have any information on. Should the button label make this clear?

---

Also, we obviously don't need to copy existing block explorers, but this design of having `<` `>` buttons next to the block number itself near the top can be nice. It's quick/easy for users to access without having to go to the bottom, and it stays in the same position if clicking through a few times.

<img width="991" alt="image" src="https://github.com/user-attachments/assets/68c52fcc-dab6-481c-977e-d252dd34b34c" />

**Comment by nloureiro** [comment]
> @nloureiro To clarify... "Next" block means, next block with an available proof? or next numerical block?
> 
> Assuming next block with an available proof; ie, the next block we have any information on. Should the button label make this clear?
> 
> Also, we obviously don't need to copy existing block explorers, but this design of having `<` `>` buttons next to the block number itself near the top can be nice. It's quick/easy for users to access without having to go to the bottom, and it stays in the same position if clicking through a few times.
> 
> <img alt="image" width="991" src="https://private-user-images.githubusercontent.com/54227730/400546800-68c52fcc-dab6-481c-977e-d252dd34b34c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzYyNDQyMDksIm5iZiI6MTczNjI0MzkwOSwicGF0aCI6Ii81NDIyNzczMC80MDA1NDY4MDAtNjhjNTJmY2MtZGFiNi00ODFjLTk3N2UtZDI1MmRkMzRiMzRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTA3VDA5NTgyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY3M2M1MGJlZTcxYWNjOTczOTQwNjhhZDQ2NjVkOTMwZjM2YzY2MTA5ZDVjYjM0YThlMDg1MDM5M2M0ZTA3YjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.a-MLkS8crXpKuiMv_Log7NlJaLMguWpjasQEbp8MM4U">

hum.... I think the navigation on the bottom is still needed, but was something like this what you had in mind?

![Screen Shot 2025-01-07 10 01 47 AM](https://github.com/user-attachments/assets/9ec7bdf5-64db-46b1-a506-73271a36d18e)

**Comment by wackerow** [comment]
Yes, sorry.. I didn't mean to imply we shouldn't place what you have at the bottom. In addition to that, a simple top navigation would be nice imo.

**Comment by nloureiro** [comment]
It's a good idea; I would say let's do it; it seems simple to implement.

**Comment by nloureiro** [comment]
with new design no longer needed
